google_drive/utils.py

```diff
@@ -1,37 +0,0 @@
-# -*- coding: utf-8 -*-
```

Comment:
Yes, but they got replaced by using the stuff in xblock-utils, basically all that was left in here was the call to getLogger, which seemed silly to have it's own file